virtcontainers/utils/utils_test.go

```diff
@@ -325,3 +325,12 @@ func TestSupportsVsocks(t *testing.T) {
 
 	assert.True(SupportsVsocks())
 }
+
+func TestValidCgroupPath(t *testing.T) {
+	assert := assert.New(t)
+
+	assert.Equal(defaultCgroupPath, ValidCgroupPath("../../../"))
```

Comment:
Nit: Might be useful to add in a few more scenarios such as `""`, `"/"`, `"../"`, `"."`, `"./../"`, `"o / m /../ g"`, *etc* ;)